src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnection.cs

```diff
@@ -61,7 +61,9 @@ internal sealed partial class HttpConnection : HttpConnectionBase, IDisposable
         private int _readOffset;
         private int _readLength;
 
+        private long _idleSinceTickCount;
         private bool _inUse;
+        private bool _detachedFromPool;
```

Comment:
How is this different from `_inUse`?

Reply:
_detachedFromPool is used in one very specific case, which is when NT auth causes us to create a new connection (and close the old one) in order to proceed with auth.